README.md

```diff
@@ -103,6 +105,26 @@ See the documentation for
 and
 [MegaLinter configuration](https://oxsecurity.github.io/megalinter/latest/configuration/).
 
+### `megalinter-incremental`
+
+Run [MegaLinter](https://oxsecurity.github.io/megalinter/) (skipping linters
```

Comment:
For consistency with the rest of the README, we only link to the tool the first time it's mentioned within a major section of the documentation (unless we are specifically linking to some part of the documentation). This will be a potential gotcha in the subsequent pull request since we'll want to add this link back upon removing the v5 hooks.